src/routes/userRouter.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const express = require('express');
+const userController = require('../controllers/userController');
+
+const userRouter = express.Router();
+
+userRouter.get('/', userController.getAllUsers);
+
+userRouter.get('/:userId', userController.getUserById);
+
+userRouter.post('/', userController.createUser);
+
+userRouter.delete('/:userId', userController.deleteUser);
+
+userRouter.patch('/:userId', userController.updateUser);
```

Comment:
Add error handling middleware to return 404 if expected entity doesn't exist and 400 if required parameter is not passed.